components/mautic/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@pipedream/mautic",
+  "version": "0.3.4",
```

Comment:
```suggestion
  "version": "0.0.1",
```

Reply:
The versioning for `package.json` is not the same as for components.

Reply:
Have a doc about the `package.json` versioning?